packages/@rnw-scripts/promote-release/package.json

```diff
@@ -23,7 +23,7 @@
     "glob": "^7.1.6",
     "js-yaml": "^3.14.0",
     "lodash": "^4.17.15",
-    "simplegit": "^1.0.2",
+    "simple-git": "^1.131.0",
```

Comment:
Not actually related, but fixing this typo which led to the wrong dep being included.